ocaml/xapi/importexport.ml

```diff
@@ -299,7 +280,12 @@ let remote_metadata_export_import ~__context ~rpc ~session_id ~remote_address ~r
            | `success -> begin
                debug "Remote metadata import succeeded";
                let result = Client.Task.get_result rpc session_id remote_task in
-               API.Legacy.From.ref_VM_set "" (Xml.parse_string result)
+               try
+                 result
+                 |> Xmlrpc.of_string
+                 |> API.ref_VM_set_of_rpc
+               with
+                 parse_error -> raise Api_errors.(Server_error (field_type_error, [Printexc.to_string parse_error]))
```

Comment:
This is a very minor point again, but the param of this error seems to the field:
```
  91   error Api_errors.field_type_error ["field"]                                                                                                                               
  92     ~doc:"The value specified is of the wrong type" ();
```

Reply:
The error is bogus. Before we were not even printing the field. 

Using the printexc I am printing the real reason we cannot unmarshal the type. It is still a field type error, but with a useful string in the metadata. I think that even if is not what announced the errors with that content are way more readable and understandable. I agree that it is anstretch, but I will keep them as they are